test/tools/router_check/router.h

```diff
@@ -41,12 +41,14 @@ struct ToolConfig {
 
   Stats::SymbolTable& symbolTable() { return *symbol_table_; }
 
-  std::unique_ptr<Http::TestHeaderMapImpl> headers_;
+  std::unique_ptr<Http::TestRequestHeaderMapImpl> request_headers_;
+  std::unique_ptr<Http::TestResponseHeaderMapImpl> response_headers_;
```

Comment:
nit: const

Reply:
I was unable to set these to const since some of the header finalization methods only take non-Const headers as parameters